virtual-server/examples/python/vsclient.py

```diff
@@ -0,0 +1,132 @@
+import os
+
+from kubernetes import client, config, watch
+
+from kubevirtclient import KubeVirtClient
+
+
+class VSClient:
+    GROUP = 'virtualservers.coreweave.com'
+    VERSION = 'v1alpha1'
+    PLURAL = 'virtualservers'
+
+    EXPECTED_CONDITIONS = {
+        'Stopped': {
+            'reason': 'VirtualServerStopped',
+            'status': 'False',
+            'type': 'Ready',
+        },
+        'Ready': {
+            'reason': 'VirtualServerReady',
+            'status': 'True',
+            'type': 'Ready',
+        }
+    }
+
+    def __init__(self, kubeconfig_path=None):
+        if kubeconfig_path is None:
+            kubeconfig_path = os.environ.get("KUBECONFIG")
```

Comment:
I believe this default path is done natively by `kube_config.load_kube_config`. No need to do it here.
Correct me if I'm wrong

https://github.com/kubernetes-client/python-base/blob/fb425a3bec5e5597ff44e810241699ba57377046/config/kube_config.py#L797